vault/identity_store.go

```diff
@@ -432,6 +432,10 @@ func mfaOktaPaths(i *IdentityStore) []*framework.Path {
 				Type:        framework.TypeBool,
 				Description: "(DEPRECATED) Use base_url instead.",
 			},
+			"use_passcode": {
+				Type:        framework.TypeBool,
+				Description: `If true, the user is reminded to use the passcode upon MFA validation. This option does not enforce using the passcode. Defaults to false.`,
+			},
```

Comment:
Mirrors the `use_passcode` field in `mfaDuoPaths`:
https://github.com/hashicorp/vault/blob/58a5b77bc6f0a5ac47e3e3d6020148b585f7213a/vault/identity_store.go#L473-L476